android/src/com/mapswithme/maps/settings/SettingsPrefsFragment.java

```diff
@@ -772,6 +773,27 @@ private void initScreenSleepEnabledPrefsCallbacks()
         });
   }
 
+  private void initShowOnLockScreenPrefsCallbacks()
+  {
+    Preference pref = findPreference(getString(R.string.pref_show_on_lock_screen));
+    if (pref == null)
```

Comment:
When is this case possible?

Reply:
I think only if there is a bug: either called before preferences have been initialized or there is no such preference in xml defined.
I copied it from other preference case

Reply:
I would prefer a clear crash if it is caused by the wrong code instead of silent return and complaining users. What does git blame say for another preference case?

Reply:
Checking for nulls was added in 3547acb16c60d24008a0d081813a1c45e1434576

Reply:
This could be related: https://github.com/mapsme/omim/pull/4558#discussion_r88602086

This check looks unnecessary for me, and it would be great to find this bug (if it's still there).

A TODO about removing this check (everywhere) could be added, or you can do it in another commit.

Reply:
I added TODO and created an issue https://github.com/organicmaps/organicmaps/issues/2049. It would be done in a separate PR. Is it ok?